pkg/packagekit/updates.jsx

```diff
@@ -581,7 +581,11 @@ class OsUpdates extends React.Component {
 
     handleLoadError(ex) {
         console.error("loading available updates failed:", JSON.stringify(ex));
-        if (ex.problem === "not-found")
+        /* Newer versions of cockpit-bridge return a proper dbus error in
```

Comment:
Could you note the actual version and the string ```HACK:``` so we can tell when to remove this?

Reply:
Done.